### PR TITLE
Changed model_sqlalchemy to ORM

### PR DESCRIPTION
**Pull request by anguye2**

### CODE REVIEW

**Review by pdx-cs-lbryan — CHANGES_REQUESTED** [review]
db_session should be retained in the gb model files. The '#noqa' at the end of the involved lines will mean that this will pass the automatic linting checks, but on your own linter it may still show up as an error. This is a conflict between sqlalchemy and flake8 where the former requires it for other files, but because it's not actually called in the gbmodel files, flake8 thinks it's an error. As a result with the change, your branch is currently causing bugs to occur

**Comment by pdx-cs-lbryan** [comment]
Is your task complete?
-removes raw sql statements from model

Is the branch caught up to master/are there merge conflicts still to be resolved?
ahead of master

Are old files unnecessarily introduced?
No

Is old code unnecessarily overwritten?
doesn't appear to be the case

Are new issues introduced to existing code? Do existing automated tests continue to work?
Log in: works
Dashboard: works
shows score range? yes
Edit:
-Can add students
-Can remove students
-can remove team
-can add team
-can remove all students from team
-changes reflected in DB
Review: can submit review
Prof Forms: Works

Report List:
**Bug encountered when generating the team reports, same issue when trying to view individual reports** (you might want to talk to Adam about this)
TypeError: 'Default MetaObject is not iterable'
seems to be coming from
report.py, line 161: for r in reports:
report.py. line 113: report = report + _make_student_report_pdf(s.id, session_id, is_final)
pdf = _make_printable_reports(session_id, team_id, is_final)

Set/Add Date: works
Automated Tests: pass

Uses 4 spaces where appropriate, not tabs?
yes

camelCase for class names, snake_case for variable and function names?
code is commented?
yes

Does the code handle edge cases? Are try/except statements used sufficiently?
No new issues added, but there's some statements in the model without try/except statements

Is there documentation on how to test your code somewhere?
no, but no new functionality

File: gbmodel/model_sqlalchemy.py [hunk 1]
```diff
@@ -318,33 +334,27 @@ def get_reports_for_student(self, student_id, term_id, is_final=None):
...
+                                          reports.is_final == is_final).first()
         else:
-            query_string = "select * from reports where reviewee=:id and tid=:term_id"
-
-        params = {'id': student_id, 'term_id': term_id, 'is_final': is_final}
-        reports = engine.execute(query_string, params)
-
+            report = reports.query.filter(report.reviewee == student_id,
+                                          report.tid == term_id).first()
         return reports
```
Comment (necrobuffalo) [inline gbmodel/model_sqlalchemy.py]: I think this should probably be report?

**Review by pdx-cs-lbryan — APPROVED** [review]

**Comment by pdx-cs-lbryan** [comment]
errors fixed
